plugins/hazelcast37/src/main/java/org/radargun/service/Hazelcast37ClientService.java

```diff
@@ -0,0 +1,57 @@
+package org.radargun.service;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.core.MemberAttributeEvent;
+import com.hazelcast.core.MembershipEvent;
+import com.hazelcast.core.MembershipListener;
+import org.radargun.Service;
+import org.radargun.config.Property;
+import org.radargun.traits.ProvidesTrait;
+
+/**
+ * Hazelcast client service
+ *
+ * @author Roman Macor &lt;rmacor@redhat.com&gt;
+ */
+
+@Service(doc = "Hazelcast client")
+public class Hazelcast37ClientService extends Hazelcast36Service {
```

Comment:
Okay, I probably misunderstood - looking into code, you start with `HazelcastClient.getInstance(...)` and use different configuration class.

But make sure that the client and embedded may independently evolve - Client service shouldn't extend Embedded service, then.